tests/sundials/arkode_03.with_sundials.geq.5.4.0.output

```diff
@@ -0,0 +1,11 @@
+0 3.9 1.1 2.8
+1 2.075274578 1.059601971 2.499947283
+2 1.100460018 1.723610111 2.499972388
+3 0.7896202372 2.326668043 2.499979776
+4 0.8117726433 2.725583965 2.499987084
+5 1.187002916 2.595541878 2.499973736
```

Comment:
SUNDIALS changed a lot of default settings and possibly also internal algorithmic details. Since the results changed only with a low relative error I thought this would be ok.

Reply:
Actually, I double-checked: In the `arkode_06` and `arkode_07` tests I get quite different results between versions 3.2.1 and 4.0.0 compared to 5.4.0. The newer version seems to have a worse accuracy. I couldn't yet figure out what exactly is causing this. I assume that some internal defaults in SUNDIALS changed, but will have a closer look.

Reply:
Thanks!

Reply:
fyi: I was able to get the results close to the original ones by choosing a interpolation mode for stiff problems. This is recommended by the authors in their own brusselator example.